docs/blog/rendered/campaign-finance.ipynb

```diff
@@ -0,0 +1,2176 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.            median_amount=_.TRANSACTION_AMT.approx_median(),
</pre><p>This might be a good place to use the <code>across</code> selector:</p><p> </p><pre class="ql-syntax" spellcheck="false">table.relabel({"TRANSACTION_AMT": "amount"}).group_by(by).agg(
    n_donations=_.count(),
    s.across(
        s.c("TRANSACTION_AMT"),
        dict(total=_.sum(), mean=_.mean(), median=_.approx_median())
    )
)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
SGTM, let's keep it as is then!